tests/integration/models/release.spec.js

```diff
@@ -276,6 +276,44 @@ describe('Release Model', function () {
 					});
 				});
 			});
+
+			describe('balena.model.release.setIsInvalidated()', function () {
+				releaseRetrievalFields.forEach((field) => {
```

Comment:
Oh I just realized that this should be INSIDE both `parallel()` calls :man_facepalming: 
Anyway, no big deal...